llvm/include/llvm/CodeGen/LiveDebugVariables.h

```diff
@@ -29,7 +29,7 @@ template <typename T> class ArrayRef;
 class LiveIntervals;
 class VirtRegMap;
 
-class LLVM_LIBRARY_VISIBILITY LiveDebugVariables : public MachineFunctionPass {
```

Comment:
Please split the LiveDebugVariables changes to another PR.

Reply:
Created. https://github.com/llvm/llvm-project/pull/88374